portfolio/src/main/webapp/comments-script.js

```diff
@@ -7,18 +7,26 @@ function init(){
         credentials = res;
         document.getElementById("loginRoute").innerHTML = res.loggedIn ? "Logout" : "Login";
         document.getElementById("loginRoute").setAttribute("href", res.loggedIn ? res.logoutUrl : res.loginUrl);
-        loadComments(10);
+        loadComments();
     })
     .catch(err => console.error(err));
 }
 
 //Retrieve data from the server and create a comment Element for each response.
-function loadComments(commentsNum){
+function loadComments(){
     document.getElementById("commentForm").style.visibility = credentials.loggedIn ? "visible" : "hidden";
-    let url = "/data?num="+commentsNum.toString();
+    const reqNum = document.getElementById("quantity").value;
+    const reqLanguage = document.getElementById("language").value;
+
+    const params = new URLSearchParams();
+    params.append('num', reqNum);
+    // params.append('languageCode', reqLanguage);
```

Comment:
Removing params since you already put them in URL?

Reply:
Ooops! Sorry about that. Resolved